OmiseSDK.xcodeproj/project.pbxproj

```diff
@@ -1558,7 +1558,6 @@
 					"@executable_path/Frameworks",
 					"@loader_path/Frameworks",
 				);
-				MARKETING_VERSION = 4.24.0;
```

Comment:
We don't need to include `MARKETING_VERSION` anymore?

Reply:
@aashishgurung I think the idea is we are replacing this version string with the version string set in `OmiseSDK/Client.swift`

Reply:
Somehow Swift Package Manager updates Info.plist file and all information about versions gone making this MARKETING_VERSION useless for the Framework.

Reply:
In the future I'll check how to arrange this in a better way (if possible) but at the moment don't want to waste the time doing it.